diff --git a/pybind11/setup_helpers.py b/pybind11/setup_helpers.py
index 0108626..cedf93f 100644
--- a/pybind11/setup_helpers.py
+++ b/pybind11/setup_helpers.py
@@ -46,7 +46,7 @@ import sys
 import tempfile
 import threading
 import warnings
-from sysconfig import _POSIX_BUILD
+import sysconfig
 
 try:
     from setuptools.command.build_ext import build_ext as _build_ext
@@ -59,7 +59,9 @@ import distutils.errors
 import distutils.ccompiler
 
 
-WIN = sys.platform.startswith("win32") and not _POSIX_BUILD
+WIN = sys.platform.startswith("win32") and not sysconfig._POSIX_BUILD \
+      and sysconfig.get_platform() != "mingw"
+print("PATCHED VERSION WIN=", WIN, sysconfig.get_platform())
 PY2 = sys.version_info[0] < 3
 MACOS = sys.platform.startswith("darwin")
 STD_TMPL = "/std:c++{}" if WIN else "-std=c++{}"
